plugins/flashrom/fu-flashrom-internal-device.c

```diff
@@ -33,6 +41,38 @@ fu_flashrom_internal_device_init (FuFlashromInternalDevice *self)
 	fu_device_add_icon (FU_DEVICE (self), "computer");
 }
 
+static void
+set_fdopss_lock_state (FuDevice *device, GError **error)
+{
+	const guint16 hsfs = flashrom_tuxedo_get_hsfs();
```

Comment:
is that the actual API call?!

Reply:
I think we should wait with that until it settles down on the flashrom side, as there definitely will be some changes there. The naming is inappropriate, for sure.